### PR TITLE
container_tests: allow sourcing ENV VARS from PR comment

### PR DESCRIPTION
**Pull request by zmiklank**
Tested here: https://github.com/zmiklank/s2i-nodejs-container/pull/1

Also, should this be documented somewhere?

### CODE REVIEW

File: .github/workflows/container-tests.yml
Comment (phracek) [inline .github/workflows/container-tests.yml:92]: Do we allow more then one [ENV:<variable>]?

Comment (phracek) [inline .github/workflows/container-tests.yml:92]: I would also check if variable [ENV:SOME_KEY:SOME_VALUE] contains both SOME_KEY and SOME_VALUE.

**Review by phracek — CHANGES_REQUESTED** [review]
What about openshift tests, we are going to skipp them?

File: .github/workflows/container-tests.yml
Comment (zmiklank) [reply .github/workflows/container-tests.yml:92]: Yeah, I should have documented it somewhere.
the [ENV:] can be used only once, but you can specify more env vars there. Example: [ENV:DEBUG=1;FAIL_FAST=3;TEST=test]

Comment (zmiklank) [reply .github/workflows/container-tests.yml:92]: Is this really needed?
Only members or maintainers of the sclorg would be able to use it anyway. I personally would add this to documentation somewhere, so these people know how to use it.
But if you insist I will rework this.

Comment (phracek) [reply .github/workflows/container-tests.yml:92]: What about if user specify `[ENV:DEBUG;FAIL_3;TEST]`?

Comment (phracek) [reply .github/workflows/container-tests.yml:92]: I am thinking if the user can specify a variable that can run some weird code or even if it can damage infrastructure.
The tests can be triggered by owners or maintainers of our repositories, so may be I am too paranoid :)

Comment (zmiklank) [reply .github/workflows/container-tests.yml:92]: That would not work. Should it be working? IMO not.

Comment (zmiklank) [reply .github/workflows/container-tests.yml:92]: Hmm, I need to think about it.

**Comment by phracek** [comment]
I would prefer to wait till testing CI is not up and running again. For sure.

**Comment by zmiklank** [comment]
OK. But this change will not appear in this PR's tests results because files of the GitHub Actions are taken from master branch code.

**Comment by zmiklank** [comment]
[test]

**Comment by zmiklank** [comment]
Is there anything I need to change in this PR in order to merge it? @pkubatrh @phracek

File: .github/workflows/container-tests.yml
Comment (phracek) [reply .github/workflows/container-tests.yml:92]: No Python please. This is ok.

**Comment by phracek** [comment]
[test-all]

**Review by phracek — APPROVED** [review]
LGTM from my point of view. Let's test it.

File: .github/workflows/container-tests.yml
Comment (hhorak) [reply .github/workflows/container-tests.yml:92]: And for the comment about documentation of semantics of the value -- we can either have a note how to define an arbitrary variable in the README.md in the root of this repo.. or as a comment here in the yaml?

Comment (zmiklank) [reply .github/workflows/container-tests.yml:92]: I vote for the comment in yaml.

**Pull request merged**
